zosia_site/bootstrap/main.tf

```diff
@@ -0,0 +1,19 @@
+provider "google" {
+  project = var.project_id
+  region  = var.tfstate_bucket_region
+}
+
+resource "random_id" "tfstate_bucket_prefix" {
+  byte_length = 8
+}
+
+resource "google_storage_bucket" "tfstate_bucket" {
+  name          = "${random_id.tfstate_bucket_prefix.hex}-tfstate-bucket"
+  location      = var.tfstate_bucket_region
+  force_destroy = false
+  storage_class = "STANDARD"
+
+  versioning {
+    enabled = true
+  }
```

Comment:
Great use of versioning in this case :+1:  It is totally needed here, but it might be worth it to add some lifecycle rule as well to prevent version from accumulation (I mean - we can probably limit ourselves to the last 100 versions or so) :wink:

Reply:
Done